mailauth/backends.py

```diff
@@ -28,7 +28,7 @@ def authenticate(self, request, token=None):
         except SignatureExpired:
             logger.warning("Token has expired.", exc_info=True)
         except BadSignature:
-            logger.exception("Malicious or corrupted login token received.")
+            logger.exception('Malicious or corrupted login token received: "%s"', token)
```

Comment:
why not f string :D?

Reply:
Because, loggers do their own string modulation. The benefit is, that Sentry will group them correctly if you let the logger do the modulation.